scripts/test_knowledge.py

```diff
@@ -0,0 +1,51 @@
+# Standard
+import operator
+
+# Third Party
+from datasets import Dataset
+from openai import OpenAI
+
+# First Party
+from src.instructlab.sdg import SDG
+from src.instructlab.sdg.default_flows import MMLUBenchFlow, SynthKnowledgeFlow
+from src.instructlab.sdg.pipeline import Pipeline
+
+# Please don't add you vLLM endpoint key here
+openai_api_key = "EMPTY"
+openai_api_base = "Add model endpoint here"
+
+client = OpenAI(
+    api_key=openai_api_key,
+    base_url=openai_api_base,
+)
+
+models = client.models.list()
+teacher_model = models.data[0].id
+router_model = models.data[1].id
```

Comment:
What is the router model? can it be the same as the teacher model? Also if they are the same then the index need to be `0`